examples/README.md

```diff
@@ -6,10 +6,10 @@ There is a preview of AKS Confidential Pods which is based on the Kata scheme.
 
 Some details of the ACI and Kata based solution are different. These include items such as how to invoke the az confcom policy tool and how various data is passed from the runtime into the container.
 
-The examples in this repo are split between the ACI based and Kata based. They are named '''aci''' for ACI **AND** AKS with "virtual nodes on Azure Container Instances" and '''aks''' for the Kata based solution.
+The examples in this repo are split between examples of how to use the Secure Key Release (SKR) and Encrypted Filesystem (ENCFS) sidecars. Currently, the ENCFS example is limited to ACI examples of how to use ENCFS with a Read-Only filesystem and a Read-Write filesystem. However, there are ACI based and Kata based examples for SKR. They are named '''aci''' for ACI **AND** AKS with "virtual nodes on Azure Container Instances" and '''aks-kata-cc''' for the Kata based solution. There is an additional example of how to use the SKR sidecar to contact an MAA endpoint using the Kata based solution in the '''maa-test-kata-cc''' directory.
```

Comment:
`They are named '''aci''' for ACI **AND** AKS with "virtual nodes on Azure Container Instances" and '''aks-kata-cc''' for the Kata based solution.`

I would split this up into two sentences. It's a little confusing as-is.

`The ACI and AKS  with "virtual nodes on Azure Container Instances" examples are named '''aci''' . The Kata-based solution for AKS is named '''aks-kata-cc'''.`

or something like that

Reply:
It might also be good to use the Bryce method of line splitting readmes by sentence, so the diffs are nicer